webserver/website/src/router/index.js

```diff
@@ -44,6 +49,8 @@ const routes = [
 
 const router = new VueRouter({
   mode: 'history',
+  linkActiveClass: 'active',
```

Comment:
Ist des beabsichtigt, dass nichts active ist, wenn man auf die Webseite kommt?

Reply:
Ja. Und das erklärt auch, warum es Verbindungen und nicht Verbindungssuche heist: /connections bezieht sich auf die Componente connectionDisplay. Also nicht die Suche, sondern die Anzeige von den gefundenen Verbindungen.

Reply:
Was hälst du davon wieder oben links TCP hinzuschreiben, und wenn man da drauf klickt, kommt man zum Inputform und wenn wenn man auf `Verbindungen` klickt, kommt man zu den Verbindungen?

Reply:
Wie  wäre es statt TCP sowas wie SUCHE reinmacht? Dann kommt genau das, was man auch erwartet hat.

Reply:
Nein, man erwartet immer, dass wenn man auf das Logo klickt, auf die Startseite kommt, des ist bei Reddit, YT, Tagesschau, bei allen so

Reply:
Jo bei uns ist mal halt immer schon auf der Startseite, weil ja auf jeder Unterseite auch das searchform ist.

Reply:
was hälst du davon, dass wenn man auf die Seite kommt nur `TCP`(oder was auch immer in Zukunft unser Name ist) dasteht, dann wenn man ne Verbindung sucht, kommt der Knopf `Verbindungen` rechts daneben noch dazu.
Und `TCP` leitet auf die Landingpage und `Verbindungen` halt zu den Verbindungen